kubernetes/samples/scripts/common/validate.sh

```diff
@@ -409,3 +412,51 @@ function validateDomainPVC {
   failIfValidationErrors
 }
 
+#
+# Function to validate the WDT model file exists
+#
+function validateWdtModelFile {
```

Comment:
The three new methods here are for integration test only.

Reply:
We resolved to leave this alone.